src/main/java/wolox/training/TrainingApplication.java

```diff
@@ -2,7 +2,11 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
 
+@EnableJpaRepositories("org.baeldung.persistence.repo")
```

Comment:
Remove this annotation, only used if you want to specify something non-standard :)